New Neuron Models edits

andrewgait opened the pull request:
- update to lab manual for PyNN 0.7 / 0.8
- added link to extension development guide from summer school
installation info page

andrewgait committed:
New Neuron Models edits

- update to lab manual for PyNN 0.7 / 0.8
- added link to extension development guide from summer school
installation info page

rowleya commented on spynnaker/4.0.0/PyNNOnSpiNNakerExtensions.md:
```diff
@@ -0,0 +1,21 @@
+# sPyNNaker C Development Environment
+If you are writing new neural models, synapse models or plasticity models, it is likely that you will need to compile C code to run on SpiNNaker.  Firstly, you must [follow the instructions](/common_pages/3.0.0/CDevelopmentForSpiNNaker.html) to install a C development environment.  If your code is then to make use of the sPyNNaker neural modelling code base (which avoids the need to rewrite much  ...
+
+1. Download the sPyNNaker source code as a [zip](https://github.com/SpiNNakerManchester/sPyNNaker/archive/3.0.0.zip) or [tar.gz](https://github.com/SpiNNakerManchester/sPyNNaker/archive/3.0.0.tar.gz)
```
I suspect these should refer to sPyNNaker 4.0.0 - though there probably isn't a zip file just now.  It could instead use "master.zip" and "master.tar.gz"

rowleya commented on spynnaker/4.0.0/PyNNOnSpiNNakerExtensions.md:
```diff
@@ -0,0 +1,21 @@
+This guide is intended to help you set up your development environment so that you can write new extensions to the implementation of SpiNNaker on PyNN.  This might include new neural models, or new connectors.  Note that this guide is not intended to help you make changes to the implementation itself, although it is conceivable that once your extension is stable, it might find its way in to the c ...
+
+# sPyNNaker C Development Environment
+If you are writing new neural models, synapse models or plasticity models, it is likely that you will need to compile C code to run on SpiNNaker.  Firstly, you must [follow the instructions](/common_pages/3.0.0/CDevelopmentForSpiNNaker.html) to install a C development environment.  If your code is then to make use of the sPyNNaker neural modelling code base (which avoids the need to rewrite much  ...
```
Does the 3.0.0 page work for this?

rowleya commented on spynnaker/4.0.0/hbp_summar_school.md:
```diff
@@ -12,4 +12,6 @@ For participants who only wish to use basic PyNN 0.75 functionality, we ask that
 Therefore, either follow [link](devenv.html) if you wish to use advanced features, or
 [link](../../spynnaker/3.0.0/PyNNOnSpinnakerInstall.html) if you wish to use basic functionality.
```
Looks like this link should also be fixed...

andrewgait committed:
Update edit

- remove references to 3.0.0 in "4.0.0" extension guide (if the
user has followed the development environment instructions 
properly, then extensions should already be installed)

rowleya commented on spynnaker/4.0.0/PyNNOnSpiNNakerExtensions.md:
```diff
@@ -0,0 +1,22 @@
+If you are writing new neural models, synapse models or plasticity models, it is likely that you will need to compile C code to run on SpiNNaker.  If you haven't already, follow the [development environment instructions](devenv.html) to do this.
+
+<!-- We'll need the following section again once the new release is out, but not right now.
+1. Download the sPyNNaker source code as a [zip](https://github.com/SpiNNakerManchester/sPyNNaker/archive/4.0.0.zip) or [tar.gz](https://github.com/SpiNNakerManchester/sPyNNaker/archive/4.0.0.tar.gz)
```
Remember this needs to be master.zip until the release of 4.0.0 ;)

andrewgait committed:
Change 4.0.0 to master

rowleya committed:
Updated development instructions

rowleya committed:
Updated for eclipse and added pynn install and links
